extensions/http/src/jolie/net/HttpProtocol.java

```diff
@@ -202,6 +203,7 @@
 		private static final String CONTENT_DISPOSITION = "contentDisposition";
 		private static final String DROP_URI_PATH = "dropURIPath";
 		private static final String CACHE_CONTROL = "cacheControl";
+		private static final String STRING_RAW_CONTENT = "stringRawContent";
```

Comment:
Me neither, and indeed this is just a solution for when we have to deal with buggy software, basically. In this case, @klag is dealing with a software that sends a "stream" of separate JSON objects through a single HTTP response (a bit of a hack). So the parsing needs to be handled manually. The application is not using application/octet-stream as content type, and instead triggers our JSON parser, which gets angry because it expects a single JSON message.

To my knowledge, sending a stream of JSON objects with application/json is just plain wrong: http://www.ietf.org/rfc/rfc4627.txt

But maybe I'm misreading the specs?

`forceRaw` does not sound too bad actually. Could `forceString` be better maybe? We're still doing the charset decoding, so it's not really "raw". (And we're not returning something with the `raw` data type.)

Could it ever make sense to return the content as a `raw` (byte array)? In that case, we may want a parameter that can take `string` and `raw` as values and then act accordingly.